packages/io.prometheus/Configuration.pkl

```diff
@@ -384,7 +397,7 @@ class RelabelConfig {
 /// * `"labeldrop"`: Match `regex` against all label names.
 ///   Any label that matches will be removed from the set of labels.
 /// * `"labelkeep"`: Match `regex` against all label names.
-typealias RelabelAction = *"replace"|"keep"|"drop"|"hashmod"|"labelmap"|"labeldrop"|"labelkeep"
+typealias RelabelAction = * "replace"|"keep"|"drop"|"hashmod"|"labelmap"|"labeldrop"|"labelkeep"
```

Comment:
```suggestion
typealias RelabelAction = *"replace"|"keep"|"drop"|"hashmod"|"labelmap"|"labeldrop"|"labelkeep"
```

Reply:
Should these have default cases set at all? The only usage is nullable so it's never being defaulted to `"replace"` meaningfully.